SetCurrentVersion.cmd

```diff
@@ -1,9 +1,9 @@
-set MAJOR_PREVIOUS=2
+set MAJOR_PREVIOUS=4
 set MINOR_PREVIOUS=0
 set PATCH_PREVIOUS=0
-set PRERELEASE_PREVIOUS=-rc2
+set PRERELEASE_PREVIOUS=-rc3
 
-set MAJOR=2
```

Comment:
>2 [](http://example.com/codeflow?start=10&length=1)

I think we should have changed to 4 before this PR.

Reply:
yes, this was missed in the previous PR, we should have changed it at that time.